### ISSUE TITLE
Feature wish: capture images while recording without loosing frames

### ISSUE THREAD

**Issue by goermezer** [silvanmelchior/RPi_Cam_Web_Interface#331]
Hi, and thanks for this wonderful project. I use it for several years now and I`m very happy with it. 3 WIFI cams are running successful without major issues so far.

But the small issue I face is: whenever a movie is recorded, and while recording an image is captured, the stream will hang for a couple of frames. The final recording simply will miss at the position where the image was captured 0,x seconds. 

I set the buffer feature to 2000ms and added to the sheduler when movement starts: im;ca 1. This makes sure, that the recording always starts 2 seconds before movement detection and at the same time it creates an image from that interesting event. I like this very much, because I can save the most importent events to OpenHAB and can remotely see, whoever visites the house.

The picamera python module supports this with the video_port and splitter_port parameters. I was using this before and it worked really good for me. So, after switching to RPi_Cam_Web_Interface, now I begin to miss it :-)

Thanks again.
Mustafa

**Comment by roberttidey** [comment]
Yes. There is a smalll disruption to the video recording stream if an image capture is requested while this is going on. Originally the interface did not allow image capture requests whilst recording was in progress. This was removed to allow the simultaneous captures as this was felt to be useful even if there was some disruption to video.

The raspimjpeg is already doing doing splitting to handle the preview, video, and vector data from the camera system. I'm not sure if it could handle additional splitting for the still image capture. It might add constraints on what image format could be used. Currently the image format is completely independent of the video format.

I will check further.
